index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@200&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/css-reset.css">
+    <link rel="stylesheet" href="styles/main.css">
+    <script src="https://kit.fontawesome.com/9b6ee7cdf7.js" crossorigin="anonymous"></script>
+    <title>Apple Clone</title>
+</head>
+<body>
+  <div class="container">
+    <nav class="top-nav">
+        <a class="top-nav-item" href="#"><i class="fa fa-apple"></i></a>
+        <a class="top-nav-item" href="#">Store</a>
+        <a class="top-nav-item" href="#">Mac</a>
+        <a class="top-nav-item" href="#">iPod</a>
+        <a class="top-nav-item" href="#">iPhone</a>
+        <a class="top-nav-item" href="#">iPad</a>
+        <a class="top-nav-item" href="#">iTunes</a>
+        <a class="top-nav-item" href="#">Support</a>
+        <div class="top-nav-item"><input type="text"><div class="search-icon"><i class="fa fa-search"></i></div></div>
+    </nav>
+
+    <main id="main-app">
+      <!--Banner section-->
+      <section class="banner">  
+        <h1>What will you verse be?</h1>
+        <p>Everyone has something to share. See how people are using iPad in their unique way to add to the world's story.</p>
+        <a href="#"><span class="video-link">Watch &quot;Your Verse&quot; film and explore the stories</span> &nbsp;<i class="fa fa-play-circle-o"></i></a>
+      </section>
+
+      <!--Promo cards section-->
+      <div class="card">
+        <a href="#"><img src="images/promo_iphone5s.jpg" alt="iPhone 5s promo"></a>
```

Comment:
try to use fewer comments when the code is descriptive, it's a good practice to have as little as possible because if in the future you change your code you may forget to change the comments.